tests/numerics/project_05.mpirun=3.with_cxx11=on.output

```diff
@@ -0,0 +1,13 @@
+
+DEAL:cg::Starting value 0.979796
+DEAL:cg::Failure step 8 value 0.000671969
```

Comment:
that's from chebyshev preconditioner, it's expected. Fill be fixed after the PR of @kronbichler .